python/grass/gunittest/testsuite/test_assertions.py

```diff
@@ -180,6 +181,18 @@ def setUpClass(cls):
     def tearDownClass(cls):
         cls.del_temp_region()
 
+    def test_assertRasterMinMax(self):
+        test_map = tempname(10)
```

Comment:
It does not matter here, but this gives a better, more robust example:

```suggestion
        test_map = append_uuid("tmp")
```

Everything is temporary here anyway. We made some effort in deleting what we create in grass.gunittest-based tests in tear down methods, but it may not be worth it. See other tests which have bigger tear down methods.